dataplicity/jsonrpc.py

```diff
@@ -250,7 +250,7 @@ def _handle_error(self, method, error):
                                 error.get('message', self.unknown_error_msg))
 
 
-if __name__ == "__main__":
+if __name__ == "__main__":  # pragma: no cover
 
     client = JSONRPC("http://127.0.0.1:8000/dataplicityapi/jsonrpc/")
```

Comment:
This is test code during development. @toudi has better tests now, so I think we could delete this block entirely.